src/App.js

```diff
@@ -42,7 +46,7 @@ const PickupCard = ({ field, day, time, address, contact }) => {
                                     <div className="level-item has-text-centered">
                                         <div>
                                             <p className="heading">Contact</p>
-                                            <p className="subtitle is-size-5-mobile">{contact}</p>
+                                            <p className="subtitle is-size-5-mobile">{contact || 'No contact'}</p>
```

Comment:
I think maybe putting 'N/A' instead of No contact sounds better. More standardized

Reply:
@tinchonapa I think "No contact" sounds a little more friendly. But I'll change it to 'N/A'.